plugins/chat/lib/chat_channel_hashtag_data_source.rb

```diff
@@ -18,18 +18,28 @@ def self.channel_to_hashtag_item(guardian, channel)
   def self.lookup(guardian, slugs)
```

Comment:
A general note I should maybe have done before, is that the semantic from difference `lookup` and `search` are not super obvious to me.

Reply:
I did describe that in the original PR. I will move the description and the difference between the two from that PR. TLDR -- lookup is to find hashtags by exact slug match for cooking + previewing, search is for finding hashtag matches with a search term.

Reply:
Hmm actually I already have comments in the code about this that explain each, how can I make this more obvious? Should I add more stuff from this PR description? https://github.com/discourse/discourse/pull/18937

Reply:
Yes I have seen comments and know the difference, I was just wondering if the name of the function itself couldn't express it better. If you think that's ok like this, Im fine with it.